fiftyone/operators/operator.py

```diff
@@ -141,6 +142,27 @@ def config(self):
         """The :class:`OperatorConfig` for the operator."""
         raise NotImplementedError("subclass must implement config")
 
+    @property
+    def has_secrets(self):
+        """Whther this operator has any secrets registered."""
+        return bool(self._secrets)
+
+    @property
+    def secrets(self):
+        """The list of secrets known to the operator, or None."""
+        return self._secrets
+
+    def register_secrets(self, secrets):
```

Comment:
doesn't this allow registering secrets that are not in the plugin definition?

Reply:
The current implementation doesn't facilitate users to register unlisted secrets

Reply:
All I did was rename the `add_secrets()` method to `register_secrets()`